### PR TITLE
Implement GUI for viewloan command

### LINKED ISSUE #39: Integrate GUI to show loan details
Related to #30 

### GUI implementation

- Add a new label/text field in`PersonCard`

### PR DESCRIPTION
**Pull request by marcus-ny**
Fixes #39 

### Changelog

Add a new `ObservableList<Loan>` inside Model
Changes to the list, triggered by `viewloan` command will render the corresponding loans in the GUI.

Add `LoanListView` similar to `PersonListView`
Add a new BooleanProperty to indicate whether last command executed was loan-related or otherwise
Depending on previous command, GUI flips between `PersonListView` and `LoanListView`

### Expected Behaviour

Prerequisite: There is a list being displayed on the screen (if previously been displayed, we take the latest one)

Command: `viewloan <index>`

Result: List of loans attached to the <index>-th person in the previously rendered Person List is displayed. Person List should not display simultaneously.

Upon typing any other command, the GUI should change back to Person List view (default).

### CODE REVIEW

File: src/main/java/seedu/address/logic/Logic.java [hunk 1]
```diff
@@ -16,10 +18,11 @@
 public interface Logic {
     /**
      * Executes the command and returns the result.
+     *
      * @param commandText The command as entered by the user.
      * @return the result of the command execution.
      * @throws CommandException If an error occurs during command execution.
-     * @throws ParseException If an error occurs during parsing.
+     * @throws ParseException   If an error occurs during parsing.
```
Comment (kjw142857) [inline src/main/java/seedu/address/logic/Logic.java:25]: Hmm... according to the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#comments), seems like single space between exception name and reason might be the way to go. Should we just leave this line as it is?

File: src/main/java/seedu/address/model/Model.java [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
+import seedu.address.model.person.Loan;
 import seedu.address.model.person.Person;
 
 /**
  * The API of the Model component.
  */
 public interface Model {
-    /** {@code Predicate} that always evaluate to true */
+    /**
+     * {@code Predicate} that always evaluate to true
```
Comment (kjw142857) [inline src/main/java/seedu/address/model/Model.java:18]:
```suggestion
     * {@code Predicate} that always evaluates to true
```
Just a minor grammar correction here!

**Review by kjw142857 — CHANGES_REQUESTED** [review]
Great work! The logic looks sound and user testing wise, the GUI looks good too. Just a few minor cosmetic nits and we should be good to go!

File: src/main/java/seedu/address/logic/Logic.java (hunk 1 above)
Comment (marcus-ny) [reply src/main/java/seedu/address/logic/Logic.java:25]: Might have been my IntelliJ auto formatter aligning it with above exceptions. I'll change the settings. Let's take note of this and sync our settings across other team members as well 👍

Comment (kjw142857) [reply src/main/java/seedu/address/logic/Logic.java]: Great, thanks!

**Review by kjw142857 — APPROVED** [review]
LGTM, thanks for implementing the GUI!

**Pull request merged**